packages/unenv-preset/tests/build.ts

```diff
@@ -0,0 +1,28 @@
+import { execSync } from "node:child_process";
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+
+// Root of `@cloudflare/unenv-preset`
+const pkgDir = path.resolve(fileURLToPath(import.meta.url), "../../..");
+// wrangler binary
+const wranglerPath = path.join(pkgDir, "../wrangler/bin/wrangler.js");
+// Base path for resolving `@cloudflare/unenv-preset` files
+const localPresetResolveBaseDir = path.join(pkgDir, "dist/src");
+// Base path for resolving `unjs/unenv` files
+const localUnenvResolveBaseDir = path.join(pkgDir, "node_modules/unenv");
+
+console.log(pkgDir);
+
+execSync(`node ${wranglerPath} deploy --dry-run --outdir ./dist/worker`, {
```

Comment:
Sorry to be dumb but why are we building a Worker here?

Reply:
That the same thing we do for `nodejs-hybrid-app` in the `"build": "wrangler deploy --dry-run --outdir=./dist",`

That is we need to serve the worker for the tests.

We have to `execSync` here to override the resolution base folders (via `WRANGLER_UNENV_RESOLVE_PATHS`).

That's a good question, I'll add a comment to the file.